src/Module/SessionRuleFactory.php

```diff
@@ -58,6 +58,13 @@ class SessionRuleFactory implements FactoryInterface
     /* @since [*next-version*] */
     use StringTranslatingTrait;
 
+    /**
+     * The duration up till which the weekly day-of-the-week repetition processing applies.
+     *
+     * @since [*next-version*]
+     */
+    const WEEKLY_REPEAT_DOTW_DURATION_THRESHOLD = 86400;
```

Comment:
Could you convert the value to a multiplication expression? This would add clarity WRT how long this is.

Reply:
Not allowed prior to 5.6

Reply:
I can document it instead.

Reply:
Too late ;P